### PR TITLE
feat: add sass support and fonts

### PR DESCRIPTION
**Pull request by chosww**

### CODE REVIEW

File: src/assets/styles/base/_base.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import "fonts";
+
+h1 {
+    font-family: $family-sans-serif;
```
Comment (greatislander) [inline src/assets/styles/base/_base.scss:4]: I'd rather not use Sass here and use CSS custom properties. Start with a single file and use imports only when necessary, like here: https://github.com/inclusive-design/acaw-cama/blob/main/src/assets/styles/app.css

Comment (chosww) [reply src/assets/styles/base/_base.scss:4]: Oh, this is just to have working example, I agree that we should only use imports when necessary.

Comment (greatislander) [reply src/assets/styles/base/_base.scss:4]: Still I don't think we should use Sass. Maybe let's discuss tomorrow.

File: eleventy.config.js [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
     eleventyConfig.addPlugin(fluidPlugin);
 
     eleventyConfig.addPassthroughCopy({
         "src/admin/config.yml": "admin/config.yml"
     });
 
-    eleventyConfig.addPassthroughCopy({
-        "src/assets": "assets"
-    });
+    eleventyConfig.addPassthroughCopy({ "src/assets": "assets" });
```
Comment (greatislander) [inline eleventy.config.js:14]: What's this for?

File: src/assets/styles/abstracts/_variables.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    /** Typography **/
+    --family-sans-serif: "Source Sans 3", sans-serif;
+    --font-weight-normal: 400;
+    --font-weight-medium: 500;
+    --font-weight-semibold: 600;
+    --font-weight-bold: 700;
+
+    /** Colours **/
+
+    --blue: hsla(194, 60%, 59%, 1);
```
Comment (greatislander) [inline src/assets/styles/abstracts/_variables.css:11]: I'd prefix colours with `--color-` for consistency. It's helpful for grouping variables.

File: src/assets/styles/base/_fonts.css [hunk 4]
```diff
@@ -3,295 +3,295 @@
     font-family: 'Raleway';
     font-style: normal;
     font-weight: 400;
-    src: url('../fonts/raleway-v34-latin-regular.woff2') format('woff2');
+    src: url('/assets/fonts/raleway-v34-latin-regular.woff2') format('woff2');
```
Comment (greatislander) [inline src/assets/styles/base/_fonts.css:6]: It's better to use a relative URL; if this isn't working we should figure out why.

**Review by greatislander — CHANGES_REQUESTED** [review]

**Comment by greatislander** [comment]
I'm noticing the linting CI job is failing. Can you look into that as well?

**Review by greatislander — APPROVED** [review]

**Pull request merged**